pkg/core/container/fmt.go

```diff
@@ -18,7 +18,7 @@ var (
 // CheckFormat conducts an initial check of the v2 container data.
 //
 // It is expected that if a container fails this test,
-// it will not be inner-ring approved.
+// it will not be approved by the inner-ring.
```

Comment:
Could you please change inner-ring to `inner ring` here? Thanks.

Reply:
Done